data/json/mapgen/house05.json

```diff
@@ -72,31 +72,31 @@
             ],
-            "place_items" : [
-                { "item" : "bed",                  "chance" : 80, "x" : [ 2, 3], "y" : [18,19] },
-                { "item" : "bed",                  "chance" : 80, "x" : [12,13], "y" : [18,19] },
-                { "item" : "livingroom",           "chance" : 30, "x" : [ 1, 8], "y" : [ 2,10] },
-                { "item" : "livingroom",           "chance" : 70, "x" : [ 1,21], "y" : [12,15] },
-                { "item" : "bedroom",              "chance" : 60, "x" : [ 1,10], "y" : [17,21] },
-                { "item" : "dresser",              "chance" : 50, "x" : [ 2, 2], "y" : [17,17] },
-                { "item" : "bedroom",              "chance" : 60, "x" : [12,21], "y" : [17,21] },
-                { "item" : "consumer_electronics", "chance" : 20, "x" : [12,21], "y" : [17,21] },
-                { "item" : "homeguns",             "chance" : 10, "x" : [ 1,10], "y" : [17,21] },
-                { "item" : "dresser",              "chance" : 50, "x" : [21,21], "y" : [21,21] },
-                { "item" : "novels",               "chance" : 20, "x" : [ 8, 8], "y" : [ 3, 5] },
-                { "item" : "novels",               "chance" : 20, "x" : [13,13], "y" : [10,12] },
-                { "item" : "cleaning",             "chance" : 70, "x" : [10,13], "y" : [ 3, 5] },
-                { "item" : "softdrugs",            "chance" : 30, "x" : [10,13], "y" : [ 3, 5] },
-                { "item" : "kitchen",              "chance" : 70, "x" : [15,21], "y" : [ 2,11] },
-                { "item" : "fridge",               "chance" : 70, "x" : [21,21], "y" : [11,11] },
-                { "item" : "oven",                 "chance" : 70, "x" : [21,21], "y" : [10,10] },
-                { "item" : "kitchen",              "chance" : 70, "x" : [21,21], "y" : [ 9, 9] },
-                { "item" : "trash",                "chance" : 70, "x" : [ 1, 3], "y" : [ 0, 0] }
+            "place_loot" : [
+                { "group" : "bed",                  "chance" : 90, "x" : [ 2, 3], "y" : [18,19], "repeat" : [1,3] },
+                { "group" : "bed",                  "chance" : 90, "x" : [12,13], "y" : [18,19], "repeat" : [1,3] },
+                { "group" : "livingroom",           "chance" : 90, "x" : [ 1, 8], "y" : [ 2,10], "repeat" : [1,4] },
+                { "group" : "livingroom",           "chance" : 90, "x" : [ 1,21], "y" : [12,15], "repeat" : [1,4] },
+                { "group" : "bedroom",              "chance" : 80, "x" : [ 1,10], "y" : [17,21], "repeat" : [1,4] },
+                { "group" : "dresser",              "chance" : 85, "x" : [ 2, 2], "y" : [17,17], "repeat" : [1,3] },
+                { "group" : "bedroom",              "chance" : 80, "x" : [12,21], "y" : [17,21], "repeat" : [1,4] },
+                { "group" : "consumer_electronics", "chance" : 50, "x" : [12,21], "y" : [17,21], "repeat" : [1,3] },
+                { "group" : "homeguns",             "chance" :  5, "x" : [ 1,10], "y" : [17,21] },
```

Comment:
I'd avoid `homeguns`, its a throwback from the old system and remains only for compatibility. You want `guns_common` instead